utils/competitionSlug.test.ts

```
@@ -0,0 +1,53 @@
+
+  it("ignora sufixo de rota depois do slug", () => {
+    expect(
+      extractCompetitionSlug(
+        "https://copa.hemocione.com.br/competition/abc/register",
+      ),
+    ).toBe("abc");
+  });
+
+  it("ignora query e hash", () => {
+    expect(
+      extractCompetitionSlug("https://copa.hemocione.com.br/competition/abc?x=1#y"),
```

ESLint

[error] Replace `"https://copa.hemocione.com.br/competition/abc?x=1#y"` with `⏎········"https://copa.hemocione.com.br/competition/abc?x=1#y",⏎······`

(prettier/prettier)